CMakeLists.txt

```diff
@@ -263,6 +263,17 @@ endif()
 
 set(CMAKE_C_VISIBILITY_PRESET hidden)
 
+set(print_msan_notice)
+if(SECP256K1_BUILD_CTIME_TESTS AND CMAKE_C_COMPILER_ID MATCHES "^(Clang|AppleClang)$")
```

Comment:
Does Apple Clang makes sense here?

Reply:
Actually, why are you hard coding compiler IDs at all? This just makes things less flexible, and the check should be enough.

Reply:
Yeah, I had the same thought. It saves a test compilation for GCC, but it's probably simpler to drop the check for clang. In https://github.com/bitcoin-core/secp256k1/pull/1517, I also added a similar check (but for "GCC or Clang", so it's much more useless). Let's just drop it.

Reply:
> Actually, why are you hard coding compiler IDs at all? This just makes things less flexible, and the check should be enough.

Thanks! The compiler ID check has been [deleted](https://github.com/bitcoin-core/secp256k1/pull/1532#issuecomment-2133405610).

Reply:
Thanks, do you want to add a commit that also removes the `$GCC` check in configure.ac, just to keep the system consistent?

Reply:
> Thanks, do you want to add a commit that also removes the `$GCC` check in configure.ac, just to keep the system consistent?

Sure. [Added](https://github.com/bitcoin-core/secp256k1/pull/1532#issuecomment-2134674954).